core/parentchain/block-importer/src/block_importer.rs

```diff
@@ -172,7 +172,13 @@ impl<
 				.execute_indirect_calls_in_extrinsics(&block, &raw_events)
 			{
 				Ok(Some(confirm_processed_parentchain_block_call)) => {
-					calls.push(confirm_processed_parentchain_block_call);
+					let opaque_call = confirm_processed_parentchain_block_call;
+					// if we have significant downtime, this mortality means we will not confirm all imported blocks
+					let mortality = GenericMortality {
+						era: Era::mortal(512, (*block.header().number()).into()),
```

Comment:
Can we define a `DEFAULT_MORTALITY` somewhere?

Which service do you mean could be down the node? Don't we just fail to send tx's than anyhow? TBH I forgot what happens in this case.

Reply:
if no validateer is operational, we will miss confirmations. with immortal, that was not the case. we flooded the chain with confirmations upon sync

Reply:
mortality preferences will be different for some cases like this one, I imagine. That's why I didn't introduce a const

Reply:
parentchain block confirmation: long living: 512
sidechain block confirmation: pretty mortal: 16
unshielding: medium mortal: 32
init: medium mortal: 64

not sure of these values are optimal, but introducing const is of limited use IMO

Reply:
Hmm, I think ultimately, we probably want to have a sync oracle such that we will not even send those extrinsics. Because the behavior now is that we will still flood the chain, but extrinsics might immediately be dropped. Then the worker doesn't need to cache and handle those unconfirmed extrinsics either.